storage/cloud-client/requirements.txt

```diff
@@ -1,2 +1,3 @@
 google-cloud-pubsub==0.38.0
-google-cloud-storage==1.13.2
+#google-cloud-storage==1.13.2
+../google-cloud-python/storage
```

Comment:
Need to have a valid requirements.txt and then pass Kokoro presubmit tests.

Reply:
Sg, should the review wait til then? 

Clarifying context: The client is not yet released. I ran tests locally and they passed.